lib/commands/web.js

```diff
@@ -245,14 +245,21 @@ helpers.getAtomsElement = function getAtomsElement (wdId) {
 };
 
 helpers.useAtomsElement = function useAtomsElement (el) {
-  if (parseInt(el, 10) < ELEMENT_OFFSET) {
-    log.debug(`Element with id '${el}' passed in for use with ` +
+  // can be ':wdc:1627938293483' by Safari
+  const regEl = new RegExp(`\\d+`).exec(el);
+  if (_.isNull(regEl)) {
+    throw new errors.UnknownError(`Cannot convert element ID for using in WD atoms: '${el}'`);
+  }
+
+  let elementid = regEl[0];
+  if (parseInt(elementid, 10) < ELEMENT_OFFSET) {
+    log.debug(`Element with id '${elementid}' passed in for use with ` +
       `atoms, but it's out of our internal scope. Adding ${ELEMENT_OFFSET}.`);
-    el = (parseInt(el, 10) + ELEMENT_OFFSET).toString();
+    el = (parseInt(elementid, 10) + ELEMENT_OFFSET).toString();
```

Comment:
how ELEMENT_OFFSET would work/is supposed to work with element IDs above? Could you please explain this logic?

Reply:
I'm still reading why the offset is needed...
I wonder there should keep `:wdc:` for atom, but I have no environment to understand more for now...

Reply:
yep, this is also my point. It does not make sense to add any changes there until we properly understand the purpose of the existing code (which seems redundant and sometimes useless to me) and check real usage scenarios.

Reply:
the point of ELEMENT_OFFSET was to make sure that element IDs did not clash between native and web contexts. this came from the fact that many implementations back in the day simply used numeric element ids like 1, 2, 3, etc...

Reply:
ah... I see

Reply:
Thanks for the explanation @jlipps. Do you think it makes sense to remove this constant now and use identifiers "as is"?

Do you also know what is the purpose of `webElementIds` array? Can we simply get rid of it?

Reply:
If none of the other drivers use simple numeric IDs anymore, I think this is safe to reuse identifiers 'as is', especially if the atoms don't return numeric IDs either. But if the atoms return numeric IDs, there could still be clashes between separate contexts. Let's say a user switches back and forth to native contexts, and each time in a web context, gets the same element IDs even though they searched for different elements. Could that be a problem?